lib/example-templates/index.html

```diff
@@ -18,15 +18,14 @@
 
       <h1>Page templates</h1>
 
-      <p>You can find these templates in the <strong>/docs/views/templates</strong> folder.</p>
+      <p>You can find these templates in the <strong>/lib/example-templates</strong> folder.</p>
```

Comment:
Maybe this should say `/lib/example-templates/templates` since they are actually in that folder, to avoid confusion?

Reply:
@anandamaryon1 good point. Maybe I should move those templates up a level into `/lib/example-templates`?

Reply:
I think it'd be neater to keep them in their own folder, so they're not mixed in with the app pages eg. `index.html`, but happy either way, if you feel otherwise.

Reply:
@anandamaryon1 I've moved them up a level in 9729500. Feels simpler. Possibly the `layout.html` in `lib/example-templates` could be removed in future (it's now identical to `app/views/layout.html`.